README.md

```diff
@@ -91,6 +91,10 @@ Optional
     # Default 10 minutes.
     ckanext.csrf_filter.token_rotation_minutes = 10
 
+    # Exempts given regex matches from token checks.
```

Comment:
Most sites will want multiple rules, so the necessary syntax should be described here.

Reply:
Added required syntax to readme